modules/weko-search-ui/weko_search_ui/static/js/weko_search_ui/app.js

```diff
@@ -116,6 +122,21 @@ function searchResCtrl($scope, $rootScope, $http, $location) {
     $rootScope.commInfoIndex = "";
   }
 
+  $rootScope.getMessage = function(messageCode) {
```

Comment:
To reuse for other Controllers in this file, please move this method to the global scope.